README.md

```diff
@@ -47,58 +45,27 @@ When building on Linux it is sometimes necessary to run:
     $ gradle wrapper
     
 Finally, turn on the "Use the new formplayer frontend" feature flag on your CommCareHQ domain
+
+### Contributing
+
+For PRs that just modify code in the Formplayer repo, submit a PR to Formplayer on a separate branch.
+
+
+#### Contributing changes to commcare
+
+Formplayer also has a dependency on the commcare-core repository. The commcare-core `master` branch is not
+stable and Formplayer uses a different branch. The submodule repo `libs/commcare` should always be pointing to
+the `formplayer` branch.
```

Comment:
So my long-term hope for formplayer would be for us to get aligned on the normal CommCare release process. Ideally I'd say we'd want to merge formplayer's stable branch into master right before we cut each release for QA at a minimum.  That seems like a good way to ensure that we're getting regular coverage of the previous releases back in each cycle.